File: aiagent7-service/src/main/java/com/shuangqi/aiagent7/model/mapper/TPermissionMapper.java
package com.shuangqi.aiagent7.model.mapper;

import com.baomidou.mybatisplus.core.mapper.BaseMapper;
import com.shuangqi.aiagent7.model.domain.TPermission;
import org.apache.ibatis.annotations.Mapper;
import org.springframework.stereotype.Repository;

/**
 * @author lishuangqi
 * @description 针对表【t_permission(系统权限表)】的数据库操作Mapper
 * @createDate 2024-12-24 10:01:31
 * @Entity generator.domain.TPermission
 */
@Mapper
public interface TPermissionMapper extends BaseMapper<TPermission> {

}

File: langhuan-server/src/main/kotlin/com/langhuan/model/mapper/TPermissionMapper.kt
package com.langhuan.model.mapper

import com.baomidou.mybatisplus.core.mapper.BaseMapper
import com.langhuan.model.domain.TPermission
import org.apache.ibatis.annotations.Mapper

/**
 * @author lishuangqi
 * @description 针对表【t_permission(系统权限表)】的数据库操作Mapper
 * @createDate 2024-12-24 10:01:31
 * @Entity generator.domain.TPermission
 */
@Mapper
interface TPermissionMapper : BaseMapper<TPermission>
